src/utils/theme.ts

```diff
@@ -1,12 +1,14 @@
-import { STORAGE_KEYS } from "./constant";
+import { STORAGE_KEYS, THEME } from "./constant";
 
 const mediaPrefersDarkScheme = window.matchMedia(
   "(prefers-color-scheme: dark)"
 );
 mediaPrefersDarkScheme.addEventListener("change", () => updateTheme());
 
 export const updateTheme = () => {
-  const currentThemeType = localStorage.getItem(STORAGE_KEYS.THEME_TYPE);
+  const currentThemeType = localStorage.getItem(STORAGE_KEYS.THEME_TYPE)
+    ? localStorage.getItem(STORAGE_KEYS.THEME_TYPE)
+    : "system";
```

Comment:
Better use `localStorage.getItem(STORAGE_KEYS.THEME_TYPE) ?? 'system'` to reduce the duplication.